framework/encode/custom_ags_wrappers.cpp

```diff
@@ -469,5 +469,17 @@ AMD_AGS_API AGSReturnCode agsDriverExtensionsDX12_SetMarker(AGSContext*
     return result;
 }
 
+AMD_AGS_API void agsInitNotSupported()
+{
+    static bool check_version = true;
+    if (check_version)
+    {
+        GFXRECON_LOG_ERROR("AGS version mismatch for GFXR capture: GFXR doesn't support this version of "
+                           "amd_ags_x64.dll. Please remove the capture DLL and rename the original DLL back.");
+
+        check_version = false;
+    }
```

Comment:
`GFXRECON_LOG_ERROR_ONCE` can be used to log an error only once. It uses the same `static bool` mechanism.

Reply:
Done